tools/depends/target/gnutls/Makefile

```diff
@@ -1,9 +1,9 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile size-max.patch 02-darwin-getentropy.patch
+DEPS= ../../Makefile.include Makefile size-max.patch 02-darwin-getentropy.patch remove-weak_import-check-for-osx.patch add-dl-as-private-lib.patch
 
 # lib name, version
 LIBNAME=gnutls
-VERSION=3.5.10
+VERSION=3.6.11.1
```

Comment:
Is this the same as gnutls-3.6.12?

Reply:
No, 3.6.12 was only released today.

Reply:
Just bumped this on an addon and just this patch needs to be refreshed: https://github.com/xbmc/inputstream.ffmpegdirect/blob/dc40ef03478c1182693901ed4b2014da8029f04d/depends/common/gnutls/03-include-libdl-if-available.patch